test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -3207,6 +3207,75 @@ public void PackCommand_MissingPackageCausesError()
             }
         }
 
+        [Fact]
+        public void PackCommand_PackJsonCorrectLibPathInNupkgWithOutputName()
+        {
+            var nugetexe = Util.GetNuGetExePath();
+
+            using (var workingDirectory = TestFileSystemUtility.CreateRandomTestFolder())
```

Comment:
Nit, make workingDirectory a subdirectory of the test folder so that the id is a consistent value